template/en/default/global/user-error.html.tmpl

```diff
@@ -675,7 +675,7 @@
   [% ELSIF error == "fieldvalue_name_too_long" %]
     [% title = "Field Value Is Too Long" %]
     The value of a field is limited to 
-    [%+ constants.FIELD_VALUE_MAX_SIZE FILTER none %] characters. 
+    [%+ constants.MAX_FIELD_VALUE_SIZE FILTER none %] characters. 
```

Comment:
I would technically prefer you submit this as a separate pull request with its own bug (or, at the very least, its own commit), and then reference it from this PR, so that we're not mixing "refactor a function" and "repair an error" in the same pass. But I'm not going to block it.